### ISSUE TITLE
Could not load library cudnn_cnn_infer64_8.dll. Error code 126

### ISSUE THREAD

**Issue by Victoria-1** [tensorflow/tensorflow#53431]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):  binary
- TensorFlow version: 2.7.0
- Python version: 3.9.8
- Installed using virtualenv? pip? conda?:  pip
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):  vs2022
- CUDA/cuDNN version:  cuda11.5/cudnn8.3
- GPU model and memory:  RTX3060TI 8G

**Describe the problem**
I try to run the model but get the such error:
![image](https://user-images.githubusercontent.com/35481439/146149453-0cb4ba2a-bc6c-42d7-8003-684f3fb9d98c.png)

but i am sure i have installed the cuda and cudnn successfully. Here is the result of testing cuda.
**
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.5\extras\demo_suite>deviceQuery.exe
deviceQuery.exe Starting...

 CUDA Device Query (Runtime API) version (CUDART static linking)

Detected 1 CUDA Capable device(s)

Device 0: "NVIDIA GeForce RTX 3060 Ti"
  CUDA Driver Version / Runtime Version          11.5 / 11.5
  CUDA Capability Major/Minor version number:    8.6
  Total amount of global memory:                 8192 MBytes (8589410304 bytes)
  (38) Multiprocessors, (128) CUDA Cores/MP:     4864 CUDA Cores
  GPU Max Clock rate:                            1665 MHz (1.66 GHz)
  Memory Clock rate:                             7001 Mhz
  Memory Bus Width:                              256-bit
  L2 Cache Size:                                 3145728 bytes
  Maximum Texture Dimension Size (x,y,z)         1D=(131072), 2D=(131072, 65536), 3D=(16384, 16384, 16384)
  Maximum Layered 1D Texture Size, (num) layers  1D=(32768), 2048 layers
  Maximum Layered 2D Texture Size, (num) layers  2D=(32768, 32768), 2048 layers
  Total amount of constant memory:               zu bytes
  Total amount of shared memory per block:       zu bytes
  Total number of registers available per block: 65536
  Warp size:                                     32
  Maximum number of threads per multiprocessor:  1536
  Maximum number of threads per block:           1024
  Max dimension size of a thread block (x,y,z): (1024, 1024, 64)
  Max dimension size of a grid size    (x,y,z): (2147483647, 65535, 65535)
  Maximum memory pitch:                          zu bytes
  Texture alignment:                             zu bytes
  Concurrent copy and kernel execution:          Yes with 1 copy engine(s)
  Run time limit on kernels:                     Yes
  Integrated GPU sharing Host Memory:            No
  Support host page-locked memory mapping:       Yes
  Alignment requirement for Surfaces:            Yes
  Device has ECC support:                        Disabled
  CUDA Device Driver Mode (TCC or WDDM):         WDDM (Windows Display Driver Model)
  Device supports Unified Addressing (UVA):      Yes
  Device supports Compute Preemption:            Yes
  Supports Cooperative Kernel Launch:            Yes
  Supports MultiDevice Co-op Kernel Launch:      No
  Device PCI Domain ID / Bus ID / location ID:   0 / 1 / 0
  Compute Mode:
     < Default (multiple host threads can use ::cudaSetDevice() with device simultaneously) >

deviceQuery, CUDA Driver = CUDART, CUDA Driver Version = 11.5, CUDA Runtime Version = 11.5, NumDevs = 1, Device0 = NVIDIA GeForce RTX 3060 Ti
Result = PASS
**

And when i search for the version of tensorflow with cuda and cudnn, i found there is not any information for tensorflow-gpu-2.7.0.
![image](https://user-images.githubusercontent.com/35481439/146150973-c7644279-a912-4884-9b68-2e42c93b7524.png)

Any suggestions? thanks!

**Provide the exact sequence of commands / steps that you executed before running into the problem**

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by sushreebarsa** [comment]
@Victoria-1 Could you please have a look at the[ build from source](https://www.tensorflow.org/install/source_windows#gpu) and let us know if it helps?Thanks!

**Comment by Victoria-1** [comment]
>
Thanks for your replying. I thought I mixed the environment. For I tried to install tensorflow and pytorch the same time, but the newest version of pytorch just supports cuda11.3.  That means I have both cuda11.5 in system and 11.3  in virtual environment. 
I uninstalled the all the packages and created a conda environment, reinstalled cuda11.0.2 toolkit and cudnn8.0.5 which support both tensorflow-gpu-2.4.0 and pytorch1.7.1 by conda. And it works now.

**Comment by Victoria-1** [comment]
> @Victoria-1 Could you please have a look at the[ build from source](https://www.tensorflow.org/install/source_windows#gpu) and let us know if it helps?Thanks!

Thanks for your replying. I thought I mixed the environment. For I tried to install tensorflow and pytorch the same time, but the newest version of pytorch just supports cuda11.3.  That means I have both cuda11.5 in system and 11.3  in virtual environment. 
I uninstalled the all the packages and created a conda environment, reinstalled cuda11.0.2 toolkit and cudnn8.0.5 which support both tensorflow-gpu-2.4.0 and pytorch1.7.1 by conda. And it works now.
